flask/app/tissue_samples.py

```diff
@@ -26,7 +26,11 @@ def get_tissue_sample(id: int):
     if user_id:
         tissue_sample = (
             models.TissueSample.query.filter_by(tissue_sample_id=id)
-            .options(contains_eager(models.TissueSample.datasets))
+            .options(
+                contains_eager(models.TissueSample.datasets),
+                contains_eager(models.TissueSample.created_by),
+                contains_eager(models.TissueSample.updated_by),
```

Comment:
These two should be `joinedload` since we don't explicitly join to the user table for these relationships below